go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -116,9 +116,10 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) cli.
 		}
 
 		Close()
-		ExitFunc(exit)
```

Comment:
This is to ensure kd terminate with a proper exit code. This is kind of workaround for the cli package that is used in kd. Once moved to cobra, will be removed.